packages/terra-site/src/examples/hookshot/HookshotExample.jsx

```diff
@@ -0,0 +1,114 @@
+  }
+
+  handleButtonClick() {
+    this.setState({ isOpen: !this.state.isOpen });
+  }
+
+  handleRequestClose() {
+    this.setState({ isOpen: false });
+  }
+
+  handleSelectChange(event) {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  handleInputChange(event) {
+    this.setState({ [event.target.name]: Number.parseFloat(event.target.value) });
+  }
+
+  render() {
+    const hookshotContent = (
+      <Hookshot.Content
+        onEsc={this.handleRequestClose}
+        onOutsideClick={this.handleRequestClose}
+        onResize={this.handleRequestClose}
+      >
+        <div style={{ height: '40px', width: '200px', backgroundColor: 'red' }}>Hookshot</div>
+      </Hookshot.Content>
+    );
+
+    return (
+      <div>
+        <form>
+          <label htmlFor={this.getId('hookshotAttachmentBehavior')}>Attachment Behavior</label>
+          <select id={this.getId('hookshotAttachmentBehavior')} name="hookshotAttachmentBehavior" value={this.state.hookshotAttachmentBehavior} onChange={this.handleSelectChange}>
+            {HookshotStandard.generateOptions(Hookshot.attachmentBehaviors)}
+          </select>
+          <br />
+          <br />
+          <NumberField
+            label="Attachment Margin in Px"
+            id={this.getId('hookshotAttachmentMargin')}
+            name="hookshotAttachmentMargin"
+            defaultValue={0}
+            value={this.state.hookshotAttachmentMargin}
+            style={{ width: '200px' }}
+            onChange={this.handleInputChange}
+          />
+          <label htmlFor={this.getId('hookshotContentAttachment')}>Content Attachment</label>
+          <select id={this.getId('hookshotContentAttachment')} name="hookshotContentAttachment" value={this.state.hookshotContentAttachment} onChange={this.handleSelectChange}>
+            {HookshotStandard.generateOptions(Hookshot.attachmentPositions)}
+          </select>
+          <br />
+          <br />
+          <label htmlFor={this.getId('hookshotTargetAttachment')}>Target Attachment</label>
+          <select id={this.getId('hookshotTargetAttachment')} name="hookshotTargetAttachment" value={this.state.hookshotTargetAttachment} onChange={this.handleSelectChange}>
+            {HookshotStandard.generateOptions(Hookshot.attachmentPositions)}
+          </select>
+          <br />
+          <br />
+        </form>
+        <Hookshot
+          attachmentBehavior={this.state.hookshotAttachmentBehavior}
+          attachmentMargin={this.state.hookshotAttachmentMargin}
+          content={hookshotContent}
+          contentAttachment={this.state.hookshotContentAttachment}
+          isEnabled
+          isOpen={this.state.isOpen}
+          targetAttachment={this.state.hookshotTargetAttachment}
+          targetRef={() => document.getElementById('hookshot-standard-button')}
+        />
+        <Button id="hookshot-standard-button" text="Hookshot Example" onClick={this.handleButtonClick} />
+        <br />
```

Comment:
Do we need all these br's? One thing about documentation websites is that developers will copy everything from them, even practices such as putting several br's in a row just for spacing.